packages/ra-core/src/util/FieldTitle.spec.tsx

```diff
@@ -82,4 +82,15 @@ describe('FieldTitle', () => {
         const { container } = render(<FieldTitle label="foo" isRequired />);
         expect(container.firstChild.textContent).toEqual('foo *');
     });
+
+    it('should return null if label is false', () => {
+        const { container } = render(<FieldTitle label={false} isRequired />);
+        expect(container.firstChild).toBeNull();
+    });
+
+    it('should return null if label is empty string', () => {
```

Comment:
This isn't the desired behavior. An empty string should lead to an empty label (not the absence of label)